internal/configs/sharding/testnet.go

```diff
@@ -82,5 +82,5 @@ var testnetReshardingEpoch = []*big.Int{
 	params.TestnetChainConfig.StakingEpoch,
 }
 
-var testnetV0 = MustNewInstance(4, 25, 25, numeric.OneDec(), genesis.TNHarmonyAccounts, genesis.TNFoundationalAccounts, testnetReshardingEpoch, TestnetSchedule.BlocksPerEpoch())
+var testnetV0 = MustNewInstance(4, 30, 25, numeric.OneDec(), genesis.TNHarmonyAccounts, genesis.TNFoundationalAccounts, testnetReshardingEpoch, TestnetSchedule.BlocksPerEpoch())
```

Comment:
why is this changed?

Reply:
It's to fix all the unit tests that use the test blockchain. We need to have slots for BLS keys so that one can send staking txs in the prestaking epoch. (previously we would get not sufficient slots for BLS keys -- hence the old TODO msgs)